extensions/blocks/podcast-player/index.js

```diff
@@ -28,7 +29,10 @@ export const title = __( 'Podcast Player', 'jetpack' );
 export const settings = {
 	title,
 	description: __( 'Select and play episodes from a single podcast.', 'jetpack' ),
-	icon: queueMusic,
+	icon: {
+		src: queueMusic,
+		foreground: getIconColor(),
```

Comment:
Let's take that opportunity to update the color here as well?
https://github.com/Automattic/jetpack/blob/update/editor-blocks-icon-color/extensions/blocks/podcast-player/index.js#L86-L87

Reply:
All set